core/modules/catalog-next/store/category/actions.ts

```diff
@@ -37,7 +37,7 @@ const actions: ActionTree<CategoryState, RootState> = {
       sort: searchQuery.sort,
       includeFields: entities.productList.includeFields,
       excludeFields: entities.productList.excludeFields,
-      size: pageSize
+      size: pageSize || 50
```

Comment:
it's exactly same with default value `pageSize = 50` so you can rollback this

Reply:
Fixed